addons/sale_project/views/product_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="inherit_id" ref="sale.product_template_form_view"/>
         <field name="arch" type="xml">
             <field name="invoice_policy" position="attributes">
-                <attribute name="invisible">type == 'service' and sale_ok == True</attribute>
+                <attribute name="invisible" separator="or" add="(type == 'service' and sale_ok == True)"/>
```

Comment:
AFAIK, the attribute are always added encapsulated in parentheses, so it's probably not necessary (doesn't really matter to keep it like that for now though)